src/sail_json_backend/json.ml

```diff
@@ -615,7 +615,16 @@ let rec string_of_sizeof_field k f =
     end
   end
 
-let json_of_field k f = "{ \"field\": \"" ^ f ^ "\", \"size\": " ^ string_of_sizeof_field k f ^ " }"
+let is_reserved_field field_name =
+  let reserved_prefix = "reserved_bits_" in
+  String.length field_name >= String.length reserved_prefix 
```

Comment:
One of the CI Actions is complaining about this line. I think you have an extraneous space at the end of the line.